### PR TITLE
KIALI-3195 UI-side changes for multi-namespace Istio deployment

### PR DESCRIPTION
**Pull request by jshaughn**
*** SERVER PR: https://github.com/kiali/kiali/pull/1321

### CODE REVIEW

File: src/config/ServerConfig.ts [hunk 1]
```diff
@@ -80,3 +81,17 @@ export const setServerConfig = (svcConfig: ServerConfig) => {
 
   computeValidDurations(serverConfig);
 };
+
+export const isIstioNamespace = (namespace: string): boolean => {
+  if (namespace === serverConfig.istioNamespace) {
+    return true;
+  }
+  if (serverConfig.istioComponentNamespaces) {
+    for (const ns of Object.values(serverConfig.istioComponentNamespaces)) {
```
Comment (josejulio) [inline src/config/ServerConfig.ts]: The following is equivalent  `Object.values(serverConfig.istioComponentNamespaces).includes(namespace)`.

Comment (mtho11) [reply src/config/ServerConfig.ts]: Anytime we can replace iteration with declaration is a good thing.

Comment (jshaughn) [reply src/config/ServerConfig.ts]: You kids just love your condensed code...  OK, I'll change it.

**Comment by jshaughn** [comment]
@josejulio @gbaufake  You don't actually have to test the multi-namespace stuff, but we need to make sure this doesn't introduce any regressions.  So please pull the server+ui PRs and smoke test the usual single-namespace deploy.  Thanks.

**Pull request merged**